📚 - NX   Unable to resolve taiga-ui:ng-add

### What is the affected URL?

https://taiga-ui.dev/getting-started

### Description

For NX project. It is not enough run `nx g taiga-ui:ng-add` to successfully add taiga-ui. You need start from installing of schematics with `npm i taiga-ui`. 

Otherwise we will get a error:

```
 nx g taiga-ui:ng-add 

 > NX   Unable to resolve taiga-ui:ng-add.

   Cannot find module 'taiga-ui/package.json'
   Require stack:
   ...
 ```


### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

--- a/projects/demo/src/modules/app/home/examples/nx-add.md
+++ b/projects/demo/src/modules/app/home/examples/nx-add.md
@@ -1,0 +1,4 @@
+```bash
+npm i taiga-ui
+nx g taiga-ui:ng-add
+```

--- a/projects/demo/src/modules/app/home/home.component.ts
+++ b/projects/demo/src/modules/app/home/home.component.ts
@@ -24,6 +24,7 @@ export class HomeComponent {
     readonly importLocalLess = import(`./examples/import-local-less.md?raw`);
     readonly main = import(`./examples/main.md?raw`);
     readonly addons = import(`./examples/addons.md?raw`);
+    readonly nxAdd = import(`./examples/nx-add.md?raw`);
 
     readonly customGlobalStyle = import(
         `../../../../../styles/taiga-ui-global.less?raw`

--- a/projects/demo/src/modules/app/home/home.template.html
+++ b/projects/demo/src/modules/app/home/home.template.html
@@ -13,7 +13,7 @@
     ></tui-doc-code>
     <tui-doc-code
         filename="Nx CLI"
-        code="nx g taiga-ui:ng-add"
+        [code]="nxAdd"
     ></tui-doc-code>
 
     <p

